JS/Lesson26/savas/script.js

```diff
@@ -6,9 +6,96 @@ p (.post-body)
 */
 
 const URL = 'https://jsonplaceholder.typicode.com/posts';
+
+document.addEventListener("DOMContentLoaded", function() {
+    
+    document.getElementById("get-posts").addEventListener("click", fetchPosts);
+  
+    document.getElementById("create-a-post").addEventListener("click", function() {
+        window.location.href = "create-post/index.html";
+      });
+  
+    document.getElementById("clear-posts").addEventListener("click", function() {
+        const postsContainer = document.getElementById("posts-container");
+        postsContainer.innerHTML = "";
+      });
+  
+    document.getElementById("submit").addEventListener("click", createANewPost);
```

Comment:
You need to delete this line since this JS file is not attached to the HTML file containing the create a new post form any more. The `Uncaught TypeError: Cannot read properties of null (reading 'addEventListener')
    at HTMLDocument.<anonymous> (script.js:23:38)` says that the element with id 'submit' is not found.

Reply:
Ok, I see. Now, we don't get any error message :)